src/cpp/client/create_channel_posix.cc

```diff
@@ -42,8 +43,10 @@ std::shared_ptr<Channel> CreateCustomInsecureChannelFromFd(
   init_lib.init();
   grpc_channel_args channel_args;
   args.SetChannelArgs(&channel_args);
-  return CreateChannelInternal("", grpc_insecure_channel_create_from_fd(
-                                       target.c_str(), fd, &channel_args));
+  return CreateChannelInternal(
+      "",
+      grpc_insecure_channel_create_from_fd(target.c_str(), fd, &channel_args),
+      nullptr);
 }
 
 #endif  // GPR_SUPPORT_CHANNELS_FROM_FD
```

Comment:
Seems like we would need an intercepted channel created from FDs as well, right?

Reply:
added